sbnana/CAFAna/XSec/Flux.cxx

```diff
@@ -130,12 +130,13 @@ namespace ana
     std::unique_ptr<TVectorD> pdg((TVectorD*)dir->Get("pdg"));
     assert(pdg && pdg->GetNrows()==1);
 
-    return std::unique_ptr<FluxTimesNuclei>(new FluxTimesNuclei(Spectrum::LoadFrom(dir, "spectrum"), (*pdg)[0]));
+    return std::unique_ptr<FluxTimesNuclei>(
+        new FluxTimesNuclei(std::move(*Spectrum::LoadFrom(dir, "spectrum")), (*pdg)[0]));
```

Comment:
Is this `move()` really necessary? It feels like it shouldn't be (the unique_ptr from LoadFrom is already a temporary)

Reply:
The compiler complains that it is an lvalue if you just dereference the unique_ptr